sema/check/__main__.py

```
@@ -0,0 +1,68 @@
+
+
+def _main(*args_list) -> bool:
+    log.info("Running sema-check with args: %s", args_list)
+    args = get_arg_parser().parse_args(args_list)
+
+    try:
+        check = make_service(args)
+        r = check.process()
+        print(f"Finished running tests: {r}")
+        return bool(r)
+    except (ValueError, IOError) as e:
```

Ruff

Replace aliased errors with `OSError`

Replace with builtin `OSError`

(UP024)